bin/build-metadata.js

```diff
@@ -331,24 +331,20 @@ function insertCountryAliases( data ) {
 }
 
 /**
- * Converts a given object to a JS string, then compiles and executes the given string and compares the two with deep
- * equality. If they are deeply equal, returns the JS string.
- * @param {{}} data
- * @returns {string}
+ * Wraps and saves data to '../client/components/phone-input/data.js'
+ * @param {Object} data
  */
-function convertToJSStringAndVerify( data ) {
-	const dataString = toJSString( data );
-	const parsedData = parseJSStringToData( dataString );
-	assert.deepEqual( parsedData , data );
-	return dataString;
-}
-
-/**
- * Wraps and saves a given JS string to '../client/components/phone-input/data.js'
- * @param dataString
- */
-function saveToFile( dataString ) {
-	var scriptStr = '// Generated by build-metadata.js\n/* eslint-disable */\nmodule.exports = ' + dataString + ';\n/* eslint-enable */\n';
+function saveToFile( data ) {
+	var scriptStr = (
+		'/** @format */\n' +
+		'// Generated by build-metadata.js\n' +
+		'/* eslint-disable */\n' +
+		Object
+			.keys( data )
+			.map( key => `export const ${ key } = ${ JSON.stringify( data[ key ], null, 3 ) };\n` )
```

Comment:
If you change `3` to `'\t'`, tabs will be used for indentation - as required by our guidelines ;)
BTW, nice idea with `JSON.stringify` 👍

Reply:
how'd that `3` get in there? should have been a `2`. doesn't really matter because `prettier` takes care of it

Reply:
tabbed in 1898bebbd